app/src/main/res/layout/tool_chooser_menu.xml

```diff
@@ -59,4 +59,14 @@
         android:onClick="chooseNewTool"
         />
 
+    <Button
+        android:id="@+id/cameraButton"
+        android:text="@string/drawCameraPicture"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_weight="1"
```

Comment:
looks good but please already add everything so that it is like the new button. Just check out develop and copy the attributes from another button and adjsut it accordingly. Will make the merge way easier if I don't have to do this.

Reply:
you'll also need to import an icon. res->drawable folder right click new->vector asset and than select an svg icon you download from the web.